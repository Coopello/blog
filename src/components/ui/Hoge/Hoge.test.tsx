import renderer from "react-test-renderer";
import { Hoge } from "./";
// ______________________________________________________
//
describe("ui/Hoge", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<Hoge />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
