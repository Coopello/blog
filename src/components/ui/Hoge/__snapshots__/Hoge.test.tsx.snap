// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/Hoge Snap Shot 1`] = `
<div>
  hoge
</div>
`;
